doc/devel/release_guide.rst

```diff
@@ -152,7 +152,7 @@ Then copy the build products into your local checkout of the
 `~/matplotlib.github.com`::
 
   cp -r build/html/* ~/matplotlib.github.com
-  cp build/latex/Matplotlib.pdf ~/matplotlib.github.com
```

Comment:
probably should leave this + where were it gets generated as capital (to not break peoples bookmarks / scripts).